### PR TITLE
aria2c: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fazlearefin** [comment]
Added new example which I believe is more useful

File: pages/common/aria2c.md [hunk 1]
```diff
@@ -32,6 +32,6 @@
 
 `aria2c --ftp-user={{username}} --ftp-passwd={{password}} "{{url}}"`
 
-- Limit download speed in bytes/s:
+- Automatically verify checksum of file upon finishing download:
```
Comment (gutjuri) [inline pages/common/aria2c.md:35]:
```suggestion
- Automatically verify the checksum of the file upon finishing the download:
```

**Review by gutjuri — COMMENTED** [review]
Thanks. However, I am not sure if we should change this, since I think that the need to limit download speed is more common than the need to verify a checksum.

**Comment by fazlearefin** [comment]
> Thanks. However, I am not sure if we should change this, since I think that the need to limit download speed is more common than the need to verify a checksum.

In that case, maybe we can find some other example which can be replaced. Most large download files (like ISO images, or even small image like firmware downloads) always come with a list of checksums. When downloading large files in splits using aria2c, there is a good chance of files getting corrupted. So I believe having the automatic ability to check the integrity of the downloaded file is an important example.

**Comment by vitorhcl** [comment]
Maybe we can instead merge the first example with the forth and add to the example description: (specify multiple URLs to download from multiple sources pointing to the same file)

**Comment by fazlearefin** [comment]
Raised a new PR https://github.com/tldr-pages/tldr/pull/12250

Closing this PR

**Pull request closed without merging**